de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/session/ProjectNegotiationObservable.java

```diff
@@ -116,4 +116,15 @@ public synchronized void remove(ProjectNegotiation negotiation) {
 
         return currentNegotiations;
     }
+
+    /**
+     * @return true if running negotiations exist
+     */
+    public synchronized boolean negotiationsInProgress() {
```

Comment:
I would give this a better name or drop it completely as calling isEmpty on the result returned by list() yields the same result.

Reply:
any name suggestion?
No!, calling isEmpty on the result of list() yields to a different result (you can / will get a list of maybe empty lists) and list() is creating a new ArrayList for every call.

Reply:
Shame on me, Sorry, I mixed it up with calling `negotiations.values().isEmpty()`.

Ok, than I change it to `isEmpty()` and replace other occurrences.